CombineAIDriver.lua

```diff
@@ -1515,3 +1506,19 @@ function CombineAIDriver:onDraw()
 	UnloadableFieldworkAIDriver.onDraw(self)
 end
 
+function CombineAIDriver:getCombineSpec(vehicle)
```

Comment:
There is already a  getCombine () function

Reply:
Apologies, I had missed that.